engine/table/src/main/java/io/deephaven/engine/table/impl/BucketedUpdateBy.java

```diff
@@ -1080,8 +1082,8 @@ private BucketedUpdateBy(@NotNull final UpdateByOperator[] operators,
             slotTracker = new UpdateBySlotTracker(control.chunkCapacityOrDefault());
 
             this.hashTable = USE_TYPED_STATE_MANAGER
-                    ? TypedHasherFactory.make(IncrementalUpdateByStateManagerTypedBase.class,
-                            keySources, keySources,
+                    ? TypedHasherFactory.make(UpdateByStateManagerTypedBase.class,
```

Comment:
Looks like we are making the same thing; so if we ditch the USE_TYPED_STATE_MANAGER parameter, we can collapse these if blocks a bit.